app/views/submissions/_form.html.erb

```diff
@@ -0,0 +1,19 @@
+<%= bootstrap_form_with url: submissions_path do |f| %>
+  <% @submission.form.form_questions.each do |form_question| %>
+    <div class="form-group mb-3">
+      <label for="response_<%= form_question.question.id %>">
+        <%= form_question.question.text %>
+      </label>
+      <input
+        type="text"
+        name="submission[<%= form_question.question.id %>]"
+        id="response_<%= form_question.question.id %>"
+        class="form-control"
+      >
+    </div>
```

Comment:
I struggled a lot with this form. I ended up abandoning Rails in favor of just interpolating with HTML to get the params in the shape I wanted. Does anyone know a more Rails-y way to do this?

Reply:
There is certainly a way -- but if this works for now. I think we can refactor.